components/Misc/S3Util.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import React from "react";
+import { Storage } from "@aws-amplify/storage";
+import { Image, ImageBackground } from "react-native";
+
+const PLACEHOLDER_IMG_URL = require("./PlaceholderImages/profile.png");
```

Comment:
Could move these to constants folder, just an idea tho not really necessary at all